packages/react/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@internationalized/date": "^3.0.0",
     "@project44-manifest/design-tokens": "^1.1.0",
+    "@radix-ui/react-slot": "^1.0.0",
```

Comment:
ohhh we are still using radix?

Reply:
Only slots, since its a good tool for cloning and passing props to children.